src/PIL/Image.py

```diff
@@ -1053,12 +1053,10 @@ def quantize(self, colors=256, method=None, kmeans=0, palette=None, dither=1):
         of colors.
 
         :param colors: The desired number of colors, <= 256
-        :param method: :data:`MEDIANCUT` (median cut),
-                       :data:`MAXCOVERAGE` (maximum coverage),
-                       :data:`FASTOCTREE` (fast octree),
-                       :data:`LIBIMAGEQUANT` (libimagequant; check support using
-                       :py:func:`PIL.features.check_feature`
-                       with ``feature="libimagequant"``).
+        :param method: :data:`MEDIANCUT` (default),
+                       :data:`MAXCOVERAGE`,
+                       :data:`FASTOCTREE` (default for RGBA images),
+                       :data:`LIBIMAGEQUANT`.
```

Comment:
Why remove the mention of `PIL.features.check_feature(feature="libimagequant")`?

Reply:
Because in the online documentation these methods link to https://pillow.readthedocs.io/en/stable/reference/Image.html#quantization-methods which already mentions exactly what it said here. I thought it would be better not to duplicate information.

I've added another commit to remove even more here and make this link to the existing section instead.

Reply:
@nulano can you have another look? Are you ok with my changes here?